src/transformStateWithClones.js

```diff
@@ -7,7 +9,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateVersions = [];
+
+  actions.reduce((currentState, action) => {
+    let nextState;
+
+    switch (action.type) {
+      case 'addProperties':
```

Comment:
Consider will be better if you create the variables for strings and use it here

Reply:
Could u please explain why i should use it? I dont really understand why it will be better after changes you sugested